src/test/mir-opt/const_allocation3/rustc.main.ConstProp.after.mir

```diff
@@ -0,0 +1,43 @@
+// MIR for `main`
+// source = MirSource { instance: Item(DefId(0:3 ~ const_allocation3[317d]::main[0])), promoted: None }
+// pass_name = ConstProp
+// disambiguator = after
+
+fn  main() -> () {
+    let mut _0: ();                      // return place in scope 0 at $DIR/const_allocation3.rs:4:11: 4:11
+    let _1: &Packed;                     // in scope 0 at $DIR/const_allocation3.rs:5:5: 5:8
+    let mut _2: &&Packed;                // in scope 0 at $DIR/const_allocation3.rs:5:5: 5:8
+
+    bb0: {
+        StorageLive(_1);                 // bb0[0]: scope 0 at $DIR/const_allocation3.rs:5:5: 5:8
+        StorageLive(_2);                 // bb0[1]: scope 0 at $DIR/const_allocation3.rs:5:5: 5:8
+        _2 = const Scalar(alloc2+0): &&Packed; // bb0[2]: scope 0 at $DIR/const_allocation3.rs:5:5: 5:8
+                                         // ty::Const
+                                         // + ty: &&Packed
+                                         // + val: Value(Scalar(alloc2+0))
+                                         // mir::Constant
+                                         // + span: $DIR/const_allocation3.rs:5:5: 5:8
+                                         // + literal: Const { ty: &&Packed, val: Value(Scalar(alloc2+0)) }
+        _1 = (*_2);                      // bb0[3]: scope 0 at $DIR/const_allocation3.rs:5:5: 5:8
+        StorageDead(_2);                 // bb0[4]: scope 0 at $DIR/const_allocation3.rs:5:8: 5:9
+        StorageDead(_1);                 // bb0[5]: scope 0 at $DIR/const_allocation3.rs:5:8: 5:9
+        _0 = ();                         // bb0[6]: scope 0 at $DIR/const_allocation3.rs:4:11: 6:2
+        return;                          // bb0[7]: scope 0 at $DIR/const_allocation3.rs:6:2: 6:2
+    }
+}
+
+alloc2 (static: const_allocation3[317d]::FOO[0], size: 8, align: 8) {
+     ├──────alloc8+0───────┤
```

Comment:
How will this look like when the text is too wide for the pointer width (quite likely on 32bit)?

Reply:
You basically get ├alloc8+024235235236536┤ even if that moves everything around and makes it hard to judge how many characters have been used up

One solution that I saw was to go back to `(0)` and `(1)` if it doesn't fit inline but make these indices *per line* and add comments at the end of the line explaining what they actually mean.

Reply:
Maybe an easier solution for now is to add something like "(4 bytes)" if the text is too wide. That will make it even wider, but well, better than no indication at all.